gokart/slack/slack_api.py

```diff
@@ -24,33 +25,27 @@ def __init__(self, token, channel: str, to_user: str) -> None:
         self._channel_id = self._get_channel_id(channel)
         self._to_user = to_user if to_user == '' or to_user.startswith('@') else '@' + to_user
 
-    def _get_channels(self, channels=[], cursor=None):
-        params = {}
-        if cursor:
-            params['cursor'] = cursor
-        response = self._client.api_call('channels.list', http_verb="GET", params=params)
-        if not response['ok']:
-            raise ChannelListNotLoadedError(f'Error while loading channels. The error reason is "{response["error"]}".')
-        channels += response.get('channels', [])
-        if not channels:
-            raise ChannelListNotLoadedError('Channel list is empty.')
-        if response['response_metadata']['next_cursor']:
-            return self._get_channels(channels, response['response_metadata']['next_cursor'])
-        else:
-            return channels
-
     def _get_channel_id(self, channel_name):
-        for channel in self._get_channels():
-            if channel['name'] == channel_name:
-                return channel['id']
-        raise ChannelNotFoundError(f'Channel {channel_name} is not found in public channels.')
+        params = {'exclude_archived': True, 'limit': 100}
+        try:
+            for channels in self._client.conversations_list(params=params):
```

Comment:
Thanks to [SlackResponse](https://github.com/slackapi/python-slackclient/blob/main/slack/web/slack_response.py), it has __next__ methond, we can remove redundant lines of 
code for pagination handling.